elevate-cpanel

```diff
@@ -5818,7 +5859,7 @@ sub _do_leapp_upgrade ($self) {
 
     my $ok = eval {
         local $ENV{LEAPP_OVL_SIZE} = 3000;
-        $self->ssystem_and_die( { keep_env => 1 }, qw{/usr/bin/leapp upgrade} );
+        $self->ssystem_and_die( { keep_env => 1 }, qw{/usr/bin/leapp upgrade --nowarn} );
```

Comment:
I am somewhat curious why this was added. Were the warnings just unhelpful noise?